src/elements/a-gltf-entity.js

```diff
@@ -0,0 +1,190 @@
+const GLTFCache = {};
```

Comment:
We may eventually want to use a `WeakMap` for this so that the cache doesn't end up retaining data for GLTF files that get unloaded. Though it would be a tradeoff if we want to load them up again.

Reply:
WeakMap keys need to be objects. Right now we use the src attribute which is a string. Idk, if we can use WeakMap. I did find a bug though. We were still using this.data as the key which is always undefined.